enable/container.py

```diff
@@ -583,44 +572,3 @@ def __components_items_changed(self, event):
     def __components_changed(self, event):
         self._layout_needed = True
         self.invalidate_draw()
-
-    # -------------------------------------------------------------------------
-    # Old / deprecated draw methods; here for backwards compatibility
-    # -------------------------------------------------------------------------
-
-    def _draw_component(self, gc, view_bounds=None, mode="normal"):
-        """ Draws the component.
-
-        This method is preserved for backwards compatibility. Overrides
-        the implementation in Component.
-        """
-        with gc:
-            gc.set_antialias(False)
-
-            self._draw_container(gc, mode)
```

Comment:
looks like this can also be removed because there are newer methods `_draw_container_*` where * is the respective layer.

Note that `Scrolled._draw` makes use of this `_draw_container` but at the moment, `_draw_container` doesn't do anything

Reply:
I haven't verified that `_draw_container` can actually be removed - that was just my hunch given the brief amount of time spent reviewing this PR. You'll want to check chaco as well and handle this in a separate PR